### ISSUE TITLE
Redesign Setup Page

### ISSUE THREAD

**Issue by enjalot** [enjalot/latent-scope#56]
The setup page is currently a scrollable list of steps that expand as the user progresses through the process. The goal is to make it more intuitive to navigate. A number of things may help this:

- [x] Each step can have its own preview (currently a single preview updates for each step)
- [x] Layout: should each step be the full page, with navigation back and forth in the process?
- [x] Use a component library for controls (give consistent UX)
- [x] #57 

A good amount of refactoring of the front-end code will be needed to support these:

- [x] Integrate [React Element Forge](https://github.com/nickgrato/react-element-forge) components
- [x] Each step has it's own self-contained component with preview

Finally, when complete we will need to update documentation

- [ ] Update [install and configure](https://enjalot.github.io/latent-scope/install-and-config) tutorial
- [ ] Update the [your first scope](https://enjalot.github.io/latent-scope/your-first-scope) tutorial

**Comment by enjalot** [comment]
Some notes from a new user's feedback:

- tried using an xlsx file, it didn't work but error wasn't clear
- not clear how to start a new dataset from setup (have to navigate home then add)
- confused on the process, expected to see clusters at the umap stage (progression is unclear)
- confused by which column is being embedded/clustered (not clear what the input is)
- unsure of how to proceed once the process is finished (not obvious to click the explore link at bottom)

**Comment by enjalot** [comment]
updating the tutorials will need to be a separate issue. Very proud of the redesign and refactor here!
